docs/DeveloperGuide.adoc

```diff
@@ -800,33 +824,134 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |user |undo my commands |reverse commands done in error
+
+|`* * *` |user |redo my commands |reverse my undos if I changed my mind
+
+|`* * *` |user |import job entries |add multiple job entries at once
+
+|`* *` |user |find job entries by the assigned employee's name |locate details of all jobs that are handled by an employee without having to go through the lists of jobs
+
+|`* *` |user |find job entries by client's name |locate details of all jobs belonging to a client without having to go through the lists of jobs
+
+|`* *` |user |find job entries by car number |locate details of all jobs related to a car number without having to go through the lists of jobs
+
+|`* *` |user |view employee's schedule on calender |easily make decision for assigning a job to an employee
+
+|`* *` |user |change the colour label of job status |easily differentiate between the job statuses
+
+|`* *` |user who likes customization |change the theme of the application |customize the theme to my preference
+
+|`* *` |user who likes customization |change the command words of the application |improve ease of use
+
+|`* *` |user |accept and reject imported changes |have control when over bulk imports
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* *` |user |modify imported job entries |import incomplete or invalid job entries
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* *` |user |add comments on job entries |record feedback to employees
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |user |sort by employees' name |sort employee list alphabetically
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* *` |user |export changes |provide feedback to employees regarding the changes
+
+|`*` |user |export completed jobs to other formats |print out all the completed job entries and view it with a better program
 |=======================================================================
 
 _{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `CarviciM` and the *Actor* is the `user`, unless specified otherwise)
+
+[discrete]
+=== Use case: Add remarks to an ongoing job entry
+
+*MSS*
+
+1.  User selects a job entry.
+2.  CarviciM shows logged remarks in the job.
+3.  User requests to log a remark.
+4.  CarviciM reflects the added remark at end of log.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The remarks list is empty.
++
+[none]
+** 2a1. CarviciM shows an empty list.
++
+Use case resumes at step 3.
+
+* 3a. The given remark is empty.
++
+Use case resumes at step 3.
+
+[discrete]
+=== Use case: Tab auto-complete commands
```

Comment:
I don't quite get this one, does it auto-complete the command word or does it help you fill up the suggested command usage too?